ManatiCR/vitaminau
# VIT-74/VIT-77/VIT-76: Add not found unit in all the forms and option all in comunity(beneficiaries).

# [VIT-74](https://manati.atlassian.net/browse/VIT-74).
Add unit catalog to the licensable technologies form.

# [VIT-77](https://manati.atlassian.net/secure/RapidBoard.jspa?rapidView=104&projectKey=VIT&modal=detail&selectedIssue=VIT-77) 
Add in unit a new option if you don't found the unit and if you select it a new text field appears with the name: "Specific Unit" in all the forms.
# [VIT-76](https://manati.atlassian.net/secure/RapidBoard.jspa?rapidView=104&projectKey=VIT&modal=detail&selectedIssue=VIT-76)
Add "todos" in comunity-beneficiaries.
## Install
- [x] ahoy drush cim -y && ahoy drush cr
## Test
- Verify in the part of beneficiaries if you select community a new option of "All", if you select "All" all provinces are selected.
- Verify in the forms the unit option, select "Don't found the unit" and check if the new fieldtext "Specific unit" appears.
### Forms to verify:
- [x] [Entrepreneurship support form](http://vitaminau.docker/formularios/apoyo-al-emprendimiento)
- [x] [Professional practice form](http://vitaminau.docker/formularios/practica-profesional)
- [x] [Cultural/ Sport proyect form](http://vitaminau.docker/formularios/proyecto-cultural-deportivo)
- [x] [Lab services form](http://vitaminau.docker/formularios/servicios-de-laboratorio)
- [x] [Technical services form](http://vitaminau.docker/formularios/servicios-tecnicos)
- [x] [TCU form](http://vitaminau.docker/formularios/trabajo-comunal-universitario)
- [x] [Work in progress form](http://vitaminau.docker/formularios/trabajo-en-curso)
- [x] [Installations use form](http://vitaminau.docker/formularios/uso-de-instalaciones)
- [x] [Sale of products form](http://vitaminau.docker/formularios/venta-de-productos)
- [x] [Licensed technologies form](http://vitaminau.docker/formularios/tecnologias-licenciables)
  - Go to the "Generadores -> Unidad" field and verify that the unit field contains the unit category (VIT-74).
## Acceptable criteria
- [x] If you select the option "Don't found the unit" a new field text opens with the name "Specific Unit".
- [x] You can't send the form if you don't fill the "Specific unit".
- [x] Option "todos" appears in all the beneficiaries if you select community.
- [x] If you select "todos" in comunity-beneficiares, all the provinces are selected.
